### PR TITLE
Starts for Open Source project of BANG

### PR DESCRIPTION
**Pull request by lucashuati**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,86 @@
...
+```
+
+## Running the tests
+```
+python manage.py test
+```
+
+## Contributing
+It's easy, create an issue or comment on an already created: I will fix/
+develop this, fork the project and submits a pull request.
```
Comment (sauloantuness) [inline README.md]: In some projects, people use a label `taken`. Maybe we can include a label too.

File: contributing.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# Contributing
```
Comment (sauloantuness) [inline contributing.md]: There are many unnecessary break lines.

File: contributing.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+## Pull Request Process
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a 
+   build.
+2. Update the README.md, if necessary, with details of changes to the interface, this includes new environment 
+   variables, exposed ports, useful file locations and container parameters.
+3. You may merge the Pull Request in once you have the sign-off of two other developers, or if you 
+   do not have permission to do that, you may request the second reviewer to merge it for you.
+
+## Code of Conduct
```
Comment (sauloantuness) [inline contributing.md]: I think it is good to reference the source of this COC. Like the **Attribution** made by this site https://dcos.io/coc/

File: contributing.md [hunk 4]
```diff
@@ -1,12 +1,10 @@
 # Contributing
```
Comment (sauloantuness) [inline contributing.md]: It is **break lines** not **blank lines**. Look for example, the line 2 to 3, or 8 to 9 etc.

Comment (lucashuati) [reply contributing.md]: The founded break lines  were removed.

**Pull request merged**